js/api.md

```diff
@@ -880,6 +881,14 @@ const client = new AWSAppSyncClient({
 });
 ```
 
+
+**Key prefix**
+
+The `AWSAppSyncClient` persists its store data (e.g. cache) using `redux-persist`. Keys in the persisted store will be prefixed by the provided `keyPrefix`.
```

Comment:
Why do we need to mention redux persist? That's an implementation detail. Also call it a cache not a store please.